logging/log_visualization/log-visualization-ocp-console.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="log-visualization-ocp-console"]
+= Log visualization with the web console
+:context: log-visualization-ocp-console
+
+toc::[]
+
+You can use the {product-title} web console to visualize log data by configuring the {log-plug}.
+
+For information about configuring the plug-in during the {logging} installation, see xref:../../logging/cluster-logging-deploying.adoc#cluster-logging-deploy-console_cluster-logging-deploying[Installing the {logging} using the web console].
```

Comment:
s/plugin/plug-in
https://redhat-documentation.github.io/supplementary-style-guide/#plugin

Reply:
The SSG was updated at the end of last year with the new guidance. You can see the announcement [here](https://groups.google.com/a/redhat.com/g/content-services-list/c/JZa535lAu9I/m/li7Xf3dcBQAJ).